separator option for .js concat

epferrari opened the pull request:
Hi, just made a small mod to respect the separator option passed to your task. Tested and working. Hope you merge, it'll save me from fixing sloppy history.js termination by hand on every freaking project ;)

sapegin commented on tasks/bower-concat.js:
```diff
@@ -35,10 +35,11 @@ module.exports = function(grunt) {
 		var done = this.async();
 		bowerMainFiles(function(jsFiles, cssFiles) {
-			concatenateAndWriteFile(jsFiles, jsDest);
+			concatenateAndWriteFile(jsFiles, jsDest, true);
```
`concatenateAndWriteFile(jsFiles, jsDest, options.separator);`

No need for an extra verbose option.

sapegin commented on tasks/bower-concat.js:
```diff
@@ -35,10 +35,11 @@ module.exports = function(grunt) {
 		var callback = this.data.callback;
 		var bowerOptions = this.data.bowerOptions || {};
 		var bowerDir = bowerOptions.relative !== false ? bower.config.cwd : '';
+		var options = this.options();
```
It should have default value.

sapegin commented:
Thanks! Could you also update the docs?

epferrari commented:
Ok, I cleaned up the verbosity and declared default option value, updated the Readme, ran the linter, and added a test case. Should be good to go. Thanks!

sapegin commented:
Thanks!

epferrari commented:
You bet. Thanks for writing the module :)
